website/docs/reference/artifacts/dbt-artifacts.md

```diff
@@ -28,6 +28,8 @@ Most dbt commands (and corresponding RPC methods) produce artifacts:
 - [catalog](catalog-json): produced by `docs generate`
 - [sources](/reference/artifacts/sources-json): produced by `source freshness`
 
+When using the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), all artifacts are downloaded by default. To skip artifacts from being downloaded, use `--download-artifacts=false` when executing a dbt command. 
```

Comment:
```suggestion
When using the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), all artifacts are downloaded by default. To skip artifacts from being downloaded, use `--download-artifacts=false` when executing a dbt command. This can help improve run-time performance but can also result in unexpected behavior so be careful. 
```